app/scripts/controllers/transactions/index.js

```diff
@@ -2605,7 +2605,7 @@ export default class TransactionController extends EventEmitter {
     );
   }
 
-  _requestApproval(txMeta) {
+  _requestApproval(txMeta, shouldShowRequest = true) {
```

Comment:
Maybe `showPopup` for clarity? And another optional options object to aid readability?

Reply:
I also think that it'd be best to keep the name used in ApprovalController in this instance. Unless we plan to change that variable there as well.